src/data/ingredientToIngredientAliases.ts

```diff
@@ -155,6 +155,21 @@ const ingredientToIngredientAliases: IngredientToIngredientAliases = {
   "Boiling Water": [],
   "Cinnamon": [],
   "Pumpkin Syrup": [],
+  "Brandy": [],
+  "Apple Cider": [],
+  "Pear Brandy": [],
+  "Hard Apple Cider": [],
+  "Dry Alsatian Riesling": [],
+  "Star Anise Pod": [],
```

Comment:
After googling, I think this one should have the alias "anise" - you tell me though.

Reply:
Agreed.